### PR TITLE
PUBDEV-8698: Fix mapr 6.2+ support

### PR DESCRIPTION
**Pull request by krasinski**

### CODE REVIEW

File: h2o-hadoop-2/h2o-mapr7.0-assembly/build.gradle [hunk 1]
```diff
@@ -0,0 +1,19 @@
+ext {
+  hadoopVersion = 'mapr7.0'
+  jetty9version = '9.4.44.v20210927'
+  hadoopMavenArtifactVersion = '2.7.6.200-eep-810'
```
Comment (mn-mikke) [inline h2o-hadoop-2/h2o-mapr7.0-assembly/build.gradle:4]: According to [this page](https://docs.datafabric.hpe.com/70/EcosystemRN/HadoopRN-2.7.4.0-2009-MEP7.0.0.html) 7.0.0 contains Hadoop 2.7.4.0-2009 and 7.0.1 Hadoop 2.7.4.100. What's the reason for referencing hadoop libraries from [Mapr 8.1](https://docs.datafabric.hpe.com/70/EcosystemRN/HadoopRN-2.7.6.200-EEP810-2201.html)?

Comment (mn-mikke) [reply h2o-hadoop-2/h2o-mapr7.0-assembly/build.gradle:4]: ok, thanks for the explanation!

**Review by mn-mikke — APPROVED** [review]
LGTM

**Review by michalkurka — APPROVED** [review]
Thank you - this looks great and I like that we can build bunch of hadoop builds specifically

**Pull request merged**